<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>800</width>
    <height>600</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>800</width>
    <height>600</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QFrame" name="frame">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>-40</y>
     <width>801</width>
     <height>641</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QFrame{
	background-color:qlineargradient(spread:pad, x1:0, y1:0, x2:1, y2:1, stop:0 rgba(0, 0, 255, 255), stop:1 rgba(255, 255, 255, 255));
}</string>
   </property>
   <property name="frameShape">
    <enum>QFrame::StyledPanel</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Raised</enum>
   </property>
   <widget class="QPushButton" name="homeButton">
    <property name="geometry">
     <rect>
      <x>650</x>
      <y>50</y>
      <width>131</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton{
	background-color: qlineargradient(spread:pad, x1:0, y1:0, x2:0.813, y2:0.880455, stop:0 rgba(255, 170, 255, 255), stop:0.153409 rgba(205, 156, 255, 255), stop:0.852273 rgba(170, 85, 255, 255));
	color: rgb(245, 235, 255);
	border-radius: 10px;
}</string>
    </property>
    <property name="text">
     <string>Home</string>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>240</x>
      <y>40</y>
      <width>251</width>
      <height>111</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background: none;</string>
    </property>
    <property name="text">
     <string>KEYWORD MANAGER</string>
    </property>
   </widget>
   <widget class="QPushButton" name="rejectButton">
    <property name="geometry">
     <rect>
      <x>570</x>
      <y>300</y>
      <width>131</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton{
	background-color: qlineargradient(spread:pad, x1:0, y1:0, x2:0.813, y2:0.880455, stop:0 rgba(255, 170, 255, 255), stop:0.153409 rgba(205, 156, 255, 255), stop:0.852273 rgba(170, 85, 255, 255));
	color: rgb(245, 235, 255);
	border-radius: 10px;
}</string>
    </property>
    <property name="text">
     <string>Reject</string>
    </property>
   </widget>
   <widget class="QPushButton" name="approveButton">
    <property name="geometry">
     <rect>
      <x>570</x>
      <y>240</y>
      <width>131</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton{
	background-color: qlineargradient(spread:pad, x1:0, y1:0, x2:0.813, y2:0.880455, stop:0 rgba(255, 170, 255, 255), stop:0.153409 rgba(205, 156, 255, 255), stop:0.852273 rgba(170, 85, 255, 255));
	color: rgb(245, 235, 255);
	border-radius: 10px;
}</string>
    </property>
    <property name="text">
     <string>Approve</string>
    </property>
   </widget>
   <widget class="QListWidget" name="keywordList">
    <property name="geometry">
     <rect>
      <x>65</x>
      <y>181</y>
      <width>401</width>
      <height>251</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background: none; 
font: 24pt &quot;.AppleSystemUIFont&quot;;</string>
    </property>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
